src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Expands all accordions and AdditionalInfo components.
+ */
+Cypress.Commands.add('expandAccordions', () => {
```

Comment:
@Mottie: And apparently by "later", I mean "now". Just couldn't resist trying it out; it works great.

Reply:
@1Copenut: I actually ended up adding the functionality to exand accordions almost right after I said I was gonig to do it later. Let me know if this looks right to you.

Reply:
@U-DON Hehe, excellent timing. I think this is on point. This method should serve us well in cases like the review page for forms where users can open all the accordions at once.

There are some one-off cases where only one accordion can be open at a time, but I think those can be handled in specific test cases.